spec/scoreCardSpec.js

```diff
@@ -0,0 +1,88 @@
+describe("Scorecard", function() {
+  var scorecard;
+
+  beforeEach(function() {
+    scorecard = new Scorecard();
+  });
+
+  it('player starts on frame 1', function() {
```

Comment:
This and the next few tests are testing the initial values of instance variables. This is not needed, and it focuses on implementation rather than behaviour.